x-pack/plugins/alerting/common/routes/rule/apis/disable/schemas/v1.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+
+export const disableRuleRequestBodySchema = schema.nullable(
```

Comment:
nitpick: we can give this entire body schema a default value so we wouldn't have to check for null in the route level